connectors/amqp-1-0-sink/2.7.1.1/amqp-1-0-sink.md

```diff
@@ -0,0 +1,383 @@
+
+You can create a configuration file (JSON or YAML) to set the following properties.
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `protocol` |String| true | "amqp" | The AMQP protocol. |
+| `host` | String| true | " " (empty string) | The AMQP service host. |
+| `port` | int |true | 5672 | The AMQP service port. |
+| `username` | String|false | " " (empty string) | The username used to authenticate to ActiveMQ. |
+| `password` | String|false | " " (empty string) | The password used to authenticate to ActiveMQ. |
+| `queue` | String|false | " " (empty string) | The queue name that messages should be read from or written to. |
+| `topic` | String|false | " " (empty string) | The topic name that messages should be read from or written to. |
+| `activeMessageType` | String|false |0 | The ActiveMQ message simple class name. |
+| `onlyTextMessage` | boolean | false | false | If set to `true`, it requires that the AMQP message type is `TextMessage`. Pulsar consumer can consume the messages with schema ByteBuffer. |
+
+**Example**
+
+* JSON 
+
+    ```json
+    {
+        "tenant": "public",
+        "namespace": "default",
+        "name": "amqp1_0-sink",
+        "inputs": ["user-op-queue-topic"],
+        "archive": "connectors/pulsar-io-amqp1_0-{version}.nar",
+        "parallelism": 1,
+        "configs":
+        {
+            "protocol": "amqp",
+            "host": "localhost",
+            "port": "5672",
+            "username": "guest",
+            "password": "guest",
+            "queue": "user-op-queue-pulsar"
+        }
+    }
+    ```
+
+* YAML
+
+    ```yaml
+    tenant: "public"
+    namespace: "default"
+    name: "amqp1_0-sink"
+    inputs: 
+      - "user-op-queue-topic"
+    archive: "connectors/pulsar-io-amqp1_0-{version}.nar"
+    parallelism: 1
+    
+    configs:
+        protocol: "amqp"
+        host: "localhost"
+        port: "5672"
+        username: "guest"
+        password: "guest"
+        queue: "user-op-queue-pulsar"
+    ```
+
+# How to use
+
+You can use the AMQP1_0 sink connector as a non built-in connector or a built-in connector as below. 
+
+## Use as non built-in connector 
+
+If you already have a Pulsar cluster, you can use the AMQP1_0 sink connector as a non built-in connector directly.
+
+This example shows how to create an AMQP1_0 sink connector on a Pulsar cluster using the command [`pulsar-admin sinks create`](http://pulsar.apache.org/tools/pulsar-admin/2.8.0-SNAPSHOT/#-em-create-em--24).
+
+
+```
+PULSAR_HOME/bin/pulsar-admin sinks create \
+--name amqp1_0-sink \
+--archive pulsar-io-amqp1_0-{version}.nar \
+--classname org.apache.pulsar.ecosystem.io.amqp.AmqpSink \
+--sink-config-file amqp-sink-config.yaml
+```
+
+## Use as built-in connector
+
+You can make the AMQP1_0 sink connector as a built-in connector and use it on standalone cluster, on-premises cluster, or K8S cluster.
```

Comment:
```suggestion
You can make the AMQP1_0 sink connector as a built-in connector and use it on the standalone cluster, on-premises cluster, or K8S cluster.
```

Reply:
here we talk about things in general, so do not use articles